druid/src/widget/added.rs

```diff
@@ -0,0 +1,61 @@
+// Copyright 2020 The Druid Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! A [`Controller`] widget that responds to [`LifeCycle::WidgetAdded`] event.
+//!
+//! [`Controller`]: struct.Controller.html
```

Comment:
Now that intra-doc-links are stable, you should be able to just do
```
//! [`Controller`]: crate::widget::Controller
```
and you'll also benefit from broken link detection.

Reply:
Ok I'll get onto this then.